tests/e2e/tests/test_smoke.py

```diff
@@ -21,6 +21,7 @@ def test_smoke(h: harness.Harness, tmp_path: Path):
 
     util.setup_k8s_snap(h, instance_id, snap_path)
     h.exec(instance_id, ["k8s", "init"])
+    util.setup_network(h, instance_id)
 
     # TODO(bschimke): The node will not report ready as the CNI is not yet implemented in the k8s snap.
```

Comment:
This is not true anymore.

As a side-note:
For more sophisticated tests we probably will need another method later. Something like:
```
util.setup_k8s(..., components=["network", "dns", ...])
```